api/services/elastigroup/azure/spotVms/paths/importFromScaleset.yaml

```diff
@@ -6,6 +6,7 @@ get:
   tags:
     - Elastigroup Azure Spot VMs
   parameters:
+    - $ref: "../../../../../commons/parameters/accountId.yaml"
```

Comment:
already fixed by the previous PR.
please align this branch with the most up-to-date master branch